src/components/quote/code/templates/blockQuoteTemplate.jsx

```diff
@@ -0,0 +1,20 @@
+var React = require('react');
+var classNames = require('classnames');
+var getComponentClasses = require('../../../../utils/getComponentClasses');
+
+module.exports = function() {
+
+  var propClasses = ['size', 'purpose', 'type'];
+  var classes = getComponentClasses('component-quote', propClasses, this.props);
+
+  return (
+      <blockquote className={classNames(classes)} cite={this.props.cite} itemScope itemType="http://schema.org/CreativeWork">
```

Comment:
Cite is correct for the html part of the blockquote. However i have realise that React doesn't support the cite attribute so i have removed this.